### PR TITLE
Background pattern on full landing tab

### PR DESCRIPTION
**Pull request by TheIsotopes**
This PR will show the background pattern on full landing tab and not only on the content_top

before this PR:

before

after this PR (looks much better):

after

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]

**Comment by KarateBrot** [comment]
Unfortunately the css styling of the background doesn't work in dark mode anymore. Can you see what's up? Probably the corresponding CSS needs to be changed slightly.

**Comment by TheIsotopes** [comment]
@KarateBrot should fixed in #3734

**Pull request merged**
